commcare_connect/templates/opportunity/visit_verification.html

```diff
@@ -103,7 +103,7 @@ <h4 class="mb-2">Flags</h4>
     </table>
     {% endif %}
     {% if visit.reason %}
-    <h4 class="mb-2">Rejection Reason</h4>
```

Comment:
not blocking, but we talked about this on the call and I am not sure we want to remove this language completely

Reply:
Fixed. [7a3bfd8](https://github.com/dimagi/commcare-connect/pull/383/commits/7a3bfd8224a27efb62e59f98028645d5a44db573)